src/index.html

```diff
@@ -22,6 +26,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="container_logo"
```

Comment:
According to the checklist, the BEM methodology should be used for class names. The class name 'container_logo' does not follow BEM convention. It should be structured as 'block__element--modifier'. In this case, 'header__logo' might be more appropriate if 'header' is the block.